RESideMenu/RESideMenuController.m

```diff
@@ -161,6 +161,15 @@ - (void)hideMenuViewController
     [self hideMenuViewControllerAnimated:YES];
 }
 
+-(UIImage *)contentViewImageSnapshot {
+    if(!_contentViewImageSnapshot) {
```

Comment:
Damn. Old habits. That's what I get for turning off clang-format in a bid to preserve existing formatting for this repo.